### DISCUSSION TITLE
Request for Assistance with Fire Simulation Modeling in Apartment Environment

### DISCUSSION THREAD

**Discussion by MohamedS2022** [firemodels/fds#12667] [Q&A]
Hello everyone,

I am working on a fire simulation scenario involving multiple objects within an apartment. The fire originates from a mini refrigerator in the kitchen and spreads to other items in the living room and bedrooms. I have the specified Heat Release Rate (HRR) curves for each object such as beds, tables, and refrigerators. However, I need assistance in modeling the fire propagation based on the ignition properties of the materials once they reach their self-ignition temperatures.

If anyone has an FDS (Fire Dynamics Simulator) file similar to this case, I would greatly appreciate your support.

Best regards,
Mohamed Sayed
PhD Student

**Comment by nsarifds** [comment]
Look at chapter 9 of the User's Guide. If you know the ignition temperature of your materials, you can define the peak HRRPUA of each material and control the burning rate with RAMP_Q (9.1.3). On Chapter 9.2 of the User Guide you can see a different approach through a pyrolysis reaction, where your materials vaporize first and then ignite. Look at the examples mentioned in the guide and pick a method that accommodates your goals. Start with one material, then introduce other components of the room and check that your numbers are what you expect them to be.
